assignment/scripts/array-practice.js

```diff
@@ -22,23 +22,23 @@ console.log('--- 2. Length of an array ---');
 console.log('Number of animals:', animalArray.length);
 
 // 2.a. TODO: Log to the console the number of foods in your array
-
+console.log('Number of favorite foods:', favoriteFoods.length);
 
 // 3. Accessing array items
 console.log('--- 3. Accessing items in an array ---');
 
 // Example: Log the first animal from the array using it's array index
-console.log('First animal is', animalArray[0])
+console.log('First animal is', animalArray[0]);
 
 // 3.a. TODO: Log the second animal in the array 
+console.log('Second animal is', animalArray[1]);
 
-
-// 3.b. TODO: Log the last animal in the array using it's array index 
-
+// 3.b. TODO: Log the last animal in the array using its array index 
+console.log('Last animal is', animalArray[3]);
 
 // 3.c. (STRETCH) TODO: Log the last animal by using the array length, 
 //      instead of the exact index number of the last item
-
+console.log('Last animal is', animalArray[animalArray.length-1]);
```

Comment:
Usually folks put spaces around operators like `-` to improve readability.

```suggestion
console.log('Last animal is', animalArray[animalArray.length - 1]);
```